### ISSUE TITLE
What hardware did you use?

### ISSUE THREAD

**Issue by kufupa** [TechCorp07/STM32F103-Code#1]
Hey there, looks like you made a brilliant project on what I assume is a fork on the Joop Brokking YMFC code. 

What hardware did u use for ur drone exactly? I.e motors, which STM32F1, etc, since I couldn't find it in your readme.

Itd also be nice if you could tell me where the nrf24l01 code is for the transmitter, since it does not seem to be in this repo.

TIA!

**Comment by TechCorp07** [comment]
yeah it was a fork from Joop, l used the KV1000 11.1V motors from china aliexpress, 30A ESCs then the stm32f103c8t6

**Comment by TechCorp07** [comment]
the Transmitter was not uploaded but you can check on youtube from electronoobs he has the guide for that code
